### PR TITLE
DOC-3723 Define the help and feedback link text differently for different books

### PR DESCRIPTION
**Pull request by nedbat**
## [DOC-3723](https://openedx.atlassian.net/browse/DOC-3723)

The help and feedback links can differ between books.  Requires 1.2.0 of the edx-theme.

### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @catong Do these look like the right URLs and labels?
- [ ] Subject matter expert: @jmbowman it's python! :)
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: shared/conf.py [hunk 1]
```diff
@@ -81,19 +81,42 @@
...
+    (PARTNER, LEARNERS): {
+        'help_url': "https://support.edx.org",
+        'help_link_text': "Contact Support",
+    },
+    (PARTNER, RESEARCHERS): {
+        'help_url': "http://edx.readthedocs.io/projects/devdata/en/latest/front_matter/preface.html#resources-for-researchers",
+    },
+    (PARTNER, DEVELOPERS): {
+        'help_url': "https://open.edx.org/getting-help",
+        'help_link_text': "Need Help?",
```
Comment (catong) [inline shared/conf.py]: @nedbat Awesome! All the places that have "Need Help?"  should be "Get Help"

Comment (catong) [inline shared/conf.py]: Add OPENEDX, LEARNERS, which I guess should have "None" for help_url and help_link_text

Comment (nedbat) [reply shared/conf.py]: done

Comment (nedbat) [reply shared/conf.py]: They would default, but explicit is better than implicit, so I've added it.

**Comment by catong** [comment]
Thanks, @nedbat 👍 for my review of the audience targets and URLs.

**Comment by catong** [comment]
FYI the single warning causing the failing Travis test is a known issue that will be resolved by https://github.com/edx/edx-documentation/pull/1545

File: shared/conf.py [hunk 2]
```diff
@@ -81,19 +81,48 @@
...
-    (PARTNER, DEVELOPERS): "https://open.edx.org/resources/e-mail-lists",
-    (OPENEDX, COURSE_TEAMS): "https://open.edx.org/resources/e-mail-lists",
-    (OPENEDX, DEVELOPERS): "https://open.edx.org/resources/e-mail-lists",
+    (PARTNER, COURSE_TEAMS): {
+        'help_url': None, #"https://partners.edx.org/forums/partner-forums",
+        'help_link_text': None,
+    },
+    (PARTNER, LEARNERS): {
+        'help_url': "https://support.edx.org",
+        'help_link_text': "Contact Support",
```
Comment (jmbowman) [inline shared/conf.py:90]: I suspect we'll want these translated at some point, but it looks like the rest of the docs are also still English only.

**Review by jmbowman — APPROVED** [review]

**Pull request merged**